src/baseconverter.js

```diff
@@ -1,18 +1,30 @@
 'use strict';
 
+
 function normalize(digitArray) {
-  while (digitArray[0] === 0) {
-    digitArray.shift();
+  const length = digitArray.length;
```

Comment:
Ha! Interesting. I noticed the base58 in the profile too. Yes, in 0.12 I
think this is used yes. Are we going to patch the old version?